paddle/fluid/framework/ir/onednn/self_attention_fuse_pass.cc

```diff
@@ -53,6 +54,14 @@ void SelfAttentionFusePass::ApplyImpl(ir::Graph* graph) const {
   LOG(WARNING) << "No-avx512 or MKL supported!";
   return;
 #endif
+
+#ifdef PADDLE_WITH_DNNL
+  if (!phi::backends::cpu::MayIUse(
+          phi::backends::cpu::cpu_isa_t::avx512_core)) {
```

Comment:
For here, the limitation is a bit strict, since in `self_dp_attention_kernel`, the commonly used ISA was `AVX512F`. While this condition will block application of machines with only `AVX512F`.